src/main/java/com/serhiiklymchuk/gmailpet/repository/UserRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.serhiiklymchuk.gmailpet.repository;
+
+import com.serhiiklymchuk.gmailpet.domain.User;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
```

Comment:
This should not be annotated as a @Repository as long as real impl bean is registered by the Spring itself.